snapfaas/src/cli_utils.rs

```diff
@@ -0,0 +1,35 @@
+use labeled::dclabel::{self, DCLabel};
```

Comment:
Documentation would be good for this file and/or functions. What do these functions do? What formats are the expecting?